libraries/engage/product/components/ProductCard/index.jsx

```diff
@@ -14,6 +14,7 @@ import {
   ProductBadges,
 } from '@shopgate/engage/product';
 import * as portals from '@shopgate/pwa-common-commerce/category';
+import { useWidgetSettings } from '@shopgate/engage/core/hooks/useWidgetSettings';
```

Comment:
please use '@shopgate/engage/core' to import 'useWidgetSettings', because there's an index file which exports among other things all hooks.